sos/report/plugins/npm.py

```diff
@@ -57,7 +57,4 @@ class NpmViaNodeJS(Npm):
     # also in Fedora 24+ it is just nodejs, no npm package
     packages = ('nodejs', )
 
-# TODO: in RHEL npm and nodejs is available via software collections
```

Comment:
This was originally added in 2016. 

If this still needs to be tracked, I can open a new issue and add a comment here.